src/event.ts

```diff
@@ -95,9 +95,7 @@ export function createEvent (req: http.IncomingMessage, res: http.ServerResponse
   // @ts-ignore
   res.res = res
   // @ts-ignore
-  res.req.res = res
-  // @ts-ignore
-  res.req.req = req
+  res.req = req
```

Comment:
Node.js standard `res` has a `req` property attached to which is a little bit different than the incoming `req`! 

I think this is originating from unenv mocks that `res.req` is unset.

Reply:
maybe we can just check if res.req exists then? (and possibly also resolve in unenv mock)

Reply:
Or we could add `res.req` polyfill if not existing :)